assets/js/frontend/user-registration-form-validator.js

```diff
@@ -354,6 +370,15 @@
 				};
 			}
 
```

Comment:
I think there is an extra `_` after `user_login_div` which may be there mistakenly.
And the condition `if (user_login_div_.length && user_login_div_.data("username-character") == "yes")` should not be independent of each other.

Reply:
The issue is resolved